src/main/java/com/fasterxml/jackson/databind/module/SimpleModule.java

```diff
@@ -312,6 +312,9 @@ protected SimpleModule setNamingStrategy(PropertyNamingStrategy naming) {
      *<p>
      * WARNING! "Last one wins" rule is applied.
      * Possible earlier addition of a serializer for a given Class will be replaced.
+     * <p>
```

Comment:
Hmmh. This is not specific to `SimpleModule` but I guess most users would find information here.
So I guess that's fine.